core/src/main/java/com/datastrato/gravitino/authorization/SupportsUserManagement.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.authorization;
+
+import com.datastrato.gravitino.exceptions.NoSuchUserException;
+import com.datastrato.gravitino.exceptions.UserAlreadyExistsException;
+import java.util.Map;
+
+/** The interface for the user management. */
+public interface SupportsUserManagement {
```

Comment:
What is the purpose of having this interface? I don't see a super necessity of having this interface.

Reply:
AccessControlManager will extends SupportsUserManagement, SupportsRoleManagement,SupportsGroupManagement. The interface will make function more clear. It will indicate the methods have the way.